tests/unit/vue-ld.spec.js

```diff
@@ -33,99 +28,130 @@ describe('VueLd Plugin', () => {
-        Using a proxy like this will allow you to return true for everything
-        not explicitly on the base object or set later.
-      */
-      flagsStub: new Proxy(
+    it('calls vueLdCallback after ready with correct context', async () => {
+      localVue.use(VueLd, {
+        ...vueLdOptions,
+        readyBeforeIdentify: false,
+      });
+      wrapper = mount(Component, {
+        localVue,
+      });
+      const vueLdCallback = jest.fn();
+      await ldClientReady(wrapper);
+      await wrapper.vm.$ld.identify(
         {
-          never: false,
-        },
-        {
-          get(obj, prop) {
-            const value = obj[prop];
-            return value === undefined ? true : value;
+          newUser: {
+            key: 'anonymous2',
+            email: 'anonymous2@test.com',
+            name: 'Anonymous User 2',
           },
-        }
-      ),
-    });
-    wrapper = mount(Component, {
-      localVue,
+        },
+        vueLdCallback
+      );
+
+      expect(vueLdCallback).toBeCalled();
+      expect(vueLdCallback.mock.instances[0]).toBe(wrapper.vm.$ld);
     });
 
-    expect(wrapper.vm.$ld.flags.never).toBe(false);
-    expect(wrapper.vm.$ld.flags.anythingElse).toBe(true);
+    it('stubs flags when passed the option', async () => {
+      localVue.use(VueLd, {
+        ...vueLdOptions,
+        /*
+          Using a proxy like this will allow you to return true for everything
+          not explicitly on the base object or set later.
+        */
+        flagsStub: new Proxy(
+          {
+            never: false,
+          },
+          {
+            get(obj, prop) {
+              const value = obj[prop];
+              return value === undefined ? true : value;
+            },
+          }
+        ),
+      });
+      wrapper = mount(Component, {
+        localVue,
+      });
+
+      expect(wrapper.vm.$ld.flags.never).toBe(false);
+      expect(wrapper.vm.$ld.flags.anythingElse).toBe(true);
 
-    wrapper.vm.$ld.flags.neverLater = false;
-    expect(wrapper.vm.$ld.flags.neverLater).toBe(false);
+      wrapper.vm.$ld.flags.neverLater = false;
+      expect(wrapper.vm.$ld.flags.neverLater).toBe(false);
 
-    delete wrapper.vm.$ld.flags.neverLater;
-    expect(wrapper.vm.$ld.flags.anythingElse).toBe(true);
+      delete wrapper.vm.$ld.flags.neverLater;
+      expect(wrapper.vm.$ld.flags.anythingElse).toBe(true);
+    });
+  });
+
+  describe('Error states', () => {
+    beforeEach(() => {
+      server.autoRespond = true;
+      server.autoRespondAfter = 0;
+      server.respondWith([
+        400,
+        { 'Content-Type': 'application/json' },
+        JSON.stringify({ message: 'Bad Request' }),
+      ]);
+    });
+
+    it.only('should set error when ldClient emits an error', async () => {
+      localVue.use(VueLd, vueLdOptions);
+      wrapper = mount(Component, {
+        localVue,
+      });
+      await ldClientReady(wrapper);
+      expect(wrapper.vm.$ld.error).toBeTruthy(undefined);
```

Comment:
nit: Why pass `undefined` to `toBeTruthy`? I didn't think that took params.

Reply:
Accident ~